### PR TITLE
Mutation toxins are no longer instant, advanced mutation toxin cure changed from frost oil to ice

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

Mutation toxins now take ~100 seconds to fully metabolize & activate
being in a cold environment (below 10 C) or consuming ice will delay the mutation
paralyze turned into knockdown on successful mutation, trying to mutate someone into their own species no longer stuns them

Advanced mutation toxin now cured with ice instead of frost oil, since frost oil is primarily a xenobio reagent (yes i know its available in botany, in one of the many plants nobody actually cares to make)

plasmaman mutation toxin no longer available in maintenance pills, neither is advanced mutation toxin, since they both suck a LOT

# Wiki Documentation

Mutation toxins now take ~100 seconds to fully metabolize & activate
being in a cold environment (below 10 C) or consuming ice will delay the mutation
paralyze turned into knockdown on successful mutation, trying to mutate someone into their own species no longer stuns them

Advanced mutation toxin now cured with ice instead of frost oil, since frost oil is primarily a xenobio reagent (yes i know its available in botany, in one of the many plants nobody actually cares to make)

# Changelog

:cl:  
tweak: species mutation toxins are no longer instant (they now take around 2 minutes), and can be counteracted by getting cold or drinking ice
tweak: slime transformation disease cure changed to ice from frost oil to make it curable
rscdel: plasmaman mutation toxin and advanced mutation toxin are no longer duplicable or present in the random reagent pool
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -524,35 +524,63 @@
...
+
+	if(data["transfurmation"] >= 50 && !already_mutating) //~100 seconds & 5 units at 2 seconds per process & 0.1 metabolization rate
+		already_mutating = TRUE
+		var/current_species = H.dna.species.type
+		var/datum/species/mutation = race
+		if(mutation && mutation != current_species) //the real mutation was the friends we made along the way :)
+			to_chat(H, span_warning("<b>You crumple in agony as your flesh wildly morphs into new forms!</b>"))
+			H.visible_message("<b>[H]</b> falls to the ground and screams as [H.p_their()] skin bubbles and froths!") //'froths' sounds painful when used with SKIN.
+			H.Knockdown(2 SECONDS)
+			addtimer(CALLBACK(src, .proc/mutate, H), 2 SECONDS)
```
Comment (tattax) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
			addtimer(CALLBACK(src, PROC_REF(mutate), H), 2 SECONDS)
```

**Comment by Moltijoe** [comment]
fun fact, this makes preterni immune to it, because this needs 50 cycles to trigger, but preterni purge any chem after 20 cycles

even beyond that, since it needs to stay in the body for the duration, doesn't that mean even a single charcoal pill will "cure" it?
hell, even with a low metabolism rate, that still requires it to not be passively metabolized before the transformation finishes

**Comment by SomeguyManperson** [comment]
> fun fact, this makes preterni immune to it, because this needs 50 cycles to trigger, but preterni purge any chem after 20 cycles

unfortunate

> even beyond that, since it needs to stay in the body for the duration, doesn't that mean even a single charcoal pill will "cure" it? hell, even with a low metabolism rate, that still requires it to not be passively metabolized before the transformation finishes

The metabolization rate was tested to work with the default 5 unit supply per slime core. Purging the chemical will, yes, also stop any transformation, so if you want to weaponize it you'll likely need to use more than the minimum dose

**Comment by Moltijoe** [comment]
since it's something that gets metabolized now
don't suppose you could add a text notification to when metabolization ends
that way people know when the charcoal has finished removing it without needing a scanner

**Comment by MajManatee** [comment]
Sounds like someone got mutated and died

**Comment by Cartlord** [comment]
i like making it not require frost oil but the rest of this is ided

**Comment by RG4ORDR** [comment]
Now you can't make funny berries to turn everyone into Felinids or Skeletons anymore?
Ided, get this PR outta here.

Mutation toxins are literally at the mercy of strange seed RNG - you literally can go weeks without ever getting a mutation toxin or you could get almost all the mutation toxins in a single round. I'll agree that the hardstun from the mutation is a tad powerful but otherwise you can't make funny plants every round like that they aren't like BS tomatoes or Deathnettles.

**Comment by MajManatee** [comment]
> Now you can't make funny berries to turn everyone into Felinids or Skeletons anymore? Ided, get this PR outta here.
> 
> Mutation toxins are literally at the mercy of strange seed RNG - you literally can go weeks without ever getting a mutation toxin or you could get almost all the mutation toxins in a single round. I'll agree that the hardstun from the mutation is a tad powerful but otherwise you can't make funny plants every round like that they aren't like BS tomatoes or Deathnettles.

Xenobio can get it every round easily and consistantly

**Comment by SomeguyManperson** [comment]
> Now you can't make funny berries to turn everyone into Felinids or Skeletons anymore? Ided, get this PR outta here.

If you read the changes, you'd know that you still can forcibly and permanently (unless xenobio wants to help, which they wont) race change people into everything wxcept plasmamen and animal slimes, and that the main thing this PR does is give counterplay to having this effectively permanent change forcibly applied by someone else, and remove its current instant stun from combat

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**